src/flir_adk_ethernet/EthernetCamera.cpp

```diff
@@ -322,7 +385,7 @@ bool EthernetCamera::closeCamera() {
 void EthernetCamera::unsetCameraEvents() {
     try {
         _pCam->UnregisterEvent(*_imageHandler);
-    } catch(Spinnaker::Exception e) {
+    } catch(const Spinnaker::Exception e) {
```

Comment:
if e is unused, just remove it (the variable name, that is - this should be `const Spinnaker::Exception &`)